src/App.test.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
+xtest('renders learn react link', () => {
```

Comment:
It might be a good idea to actually write some simple tests - it would look especially good if someone takes a look at the code.

Reply:
The example of test idea: 
- check if the modal is working correctly, if it's opening on button click,closing on X button click